pages/content/amp-dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting.md

```diff
@@ -205,7 +205,8 @@ Python-Markdown has some limitations. Use the following syntax when including co
 </div>
 
 ## Preview code samples
-You can let a code sample have a preview or a link to open the code sample in the [AMP Playground](https://playground.amp.dev/):
+
+You can let a code sample have a preview and/or a link to open the code sample in the [AMP Playground](https://playground.amp.dev/):
```

Comment:
Nit - update to:
Code samples can have a previewer and/or link to the sample in the AMP Playground